battle-service/src/main/java/by/ibondarau/tictactoe/battleservice/model/Move.java

```diff
@@ -0,0 +1,39 @@
+package by.ibondarau.tictactoe.battleservice.model;
+
+import lombok.Data;
+import lombok.experimental.Accessors;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.FetchType;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.JoinColumn;
+import javax.persistence.ManyToOne;
+import javax.persistence.SequenceGenerator;
+import javax.persistence.Table;
+import java.time.Instant;
+
+@Entity
+@Table(name = "move")
+@SequenceGenerator(name = "move_sequence", sequenceName = "move_sequence")
+@Data
+@Accessors(chain = true)
+public class Move {
+    @Id
+    @GeneratedValue(strategy = GenerationType.SEQUENCE, generator = "move_sequence")
```

Comment:
usually uuid's id's using in distributed systems  instead of numeric sequences. 

you can also extract common parameters to abstract BaseEntity . 
this entity could include id, createdAt,modifiedAt, modifiedBy,createdBy fields